setup/hr_family/odoo/__init__.py

```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
@GillesTephaneMeyomesse Can you remove the directory "setup" will be regenerated after merge...